### PR TITLE
new project wizard page

### PR DESCRIPTION
**Pull request by PeterPetrik**
Screenshot 2024-02-17 at 17 08 09

Screenshot 2024-02-17 at 16 41 17

### CODE REVIEW

File: app/qml/project/MMProjectWizardPage.qml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    id: header
+    width: projectWizardPanel.width
+    color: __style.lightGreenColor
+    title: qsTr("Create Project")
+
+    onBackClicked: {
+      projectWizardPanel.backClicked()
+    }
+  }
+
```
Comment (VitorVieiraZ) [inline app/qml/project/MMProjectWizardPage.qml:67]: 40 of margins between "Create Project" header and the rest of the content.

File: app/qml/project/MMProjectWizardPage.qml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          widgetList: projectWizardPanel.widgetsModel
+          onRemoveClicked: function( index ) {
+            fieldsModel.removeField(index)
+          }
+        }
+
+        footer: MMLink {
+              text: qsTr("Add field")
+              anchors.centerIn: parent
+              rightIcon: __style.addIcon
```
Comment (VitorVieiraZ) [inline app/qml/project/MMProjectWizardPage.qml:122]: anchors.topMargin: 20 * __dp ?

File: app/qml/project/MMProjectPanel.qml [hunk 3]
```diff
@@ -241,6 +241,7 @@ Item {
         onBackClicked: root.hidePanel()
```
Comment (VitorVieiraZ) [inline app/qml/project/MMProjectPanel.qml:242]: 20 of margins between "Home" header and the following content

**Comment by PeterPetrik** [comment]
I will fix the issues in separate PR

**Pull request merged**